src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -61,9 +62,10 @@ public void storeMessage(Message message) {
         UUID id = UUID.fromString(idString);
         String user = (String) entity.getProperty("user");
         String text = (String) entity.getProperty("text");
+	      float sentimentScore = ((Double) entity.getProperty("sentimentScore")).floatValue();
```

Comment:
nit: you have extra indentation here

Reply:
I just switched from using command-line vim for this to using a real text editor. The surrounding lines are 4 spaces + 4 spaces, this one is 1 tab + 4 spaces from the changeover. I'll fix that now.